guidelines/waic_link.js

```diff
@@ -0,0 +1,27 @@
+const trNote = "この文書内にあるリンクのうち、「Understanding WCAG 2.2」へのリンクについては、WAIC の公開する日本語版にリンク先を追加しています。WAIC の日本語訳は、 W3C の公開する英語版より内容が古い可能性がありますのでご注意ください。";
+const jaLinkText = '[日本語訳]';
+const jaLinkTitleSuffix = 'の日本語訳';
+const taregtUrlString = ['Understanding', 'Techniques'];
```

Comment:
このままだと Techniques にも日本語訳のリンクが出てしまいます。
実際、「WCAG 2.2関連文書」というところにあるTechniquesの紹介のところに日本語訳リンクが出現しているはずです。
若干キモイですが、Techniquesのトップの翻訳が終わるまでは、いったんこうしておく感じかなと。

```suggestion
const taregtUrlString = ['Understanding'];
```